src/shared/bus-wait-for-jobs.c

```diff
@@ -346,3 +350,7 @@ int bus_wait_for_jobs_one(BusWaitForJobs *d, const char *path, WaitJobsFlags fla
 
         return bus_wait_for_jobs(d, flags, extra_args);
 }
+
+void bus_wait_for_jobs_set_bus_own(BusWaitForJobs *d, bool own) {
```

Comment:
hmm, i think usually when we talk about "own" we just talk about references. But this is about closing the connection, hence I'd probably just name this bus_wait_for_jobs_set_bus_close() too, seems to make things clearer to me